mitodl/ocw-hugo-themes
# swap out chromium for google chrome and firefox in playwright tests

# What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1279

# Description (What does it do?)
The "Transcripts start time matches video start time" test in `video.spec.ts` loads a video that started refusing to play in the Chromium browser, which Playwright uses on the backend to run its tests. This PR switches out the Chromium browser for Google Chrome and Firefox, which seem to run the test fine.

# How can this be tested?
 - Check out this branch locally
 - Make sure you don't have any values in your `.env` that would cause the end to end tests to fail. It may be best to back up your .env and run the tests with an empty one so it uses the defaults.
 - Run `yarn test:e2e` and verify that the tests pass
